### PR TITLE
Add <MessageCommerce /> tests and improve component

### PR DESCRIPTION
**Pull request by vicnicius**
## Description of the pull request

Moving on with adding tests for message components, This PR:

- Adds tests for MessageCommerce tests.
- Enable ESLint on MessageCommerce component.
- Enable Typescript check on MessageCommerce component.

~Should be merged after: https://github.com/GetStream/stream-chat-react/pull/295~ ✅

### CODE REVIEW

File: src/components/Message/__tests__/MessageCommerce.test.js [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+          {},
+        );
+      } else {
+        expect(AvatarMock).not.toHaveBeenCalled();
+      }
+    },
+  );
+
+  it('should show the reaction list when message has no text', async () => {
+    const bobReaction = {
```
Comment (vishalnarkhede) [inline src/components/Message/__tests__/MessageCommerce.test.js:202]: Maybe we can have a generator function for reaction?!! We will need it at other places as well

File: src/components/Message/MessageCommerce.js [hunk 2]
```diff
@@ -200,7 +218,78 @@ class MessageCommerce extends PureComponent {
...
+    if (!message) {
+      return null;
+    }
+
+    return (
+      <div className="str-chat__message-commerce-text">
+        <div
+          data-testid="message-commerce-text-inner-wrapper"
+          className={`str-chat__message-commerce-text-inner
+									${hasAttachment ? 'str-chat__message-commerce-text-inner--has-attachment' : ''}
```
Comment (vishalnarkhede) [inline src/components/Message/MessageCommerce.js:245]: indentation ... something looks wrong

File: src/components/Message/__tests__/MessageCommerce.test.js (hunk 1 above)
Comment (vicnicius) [reply src/components/Message/__tests__/MessageCommerce.test.js:202]: Yeah... I'm using it in multiple places, indeed. I can make a separate PR and replace it everywhere if that's fine (and if no one does it before).

File: src/components/Message/MessageCommerce.js (hunk 2 above)
Comment (vicnicius) [reply src/components/Message/MessageCommerce.js]: Should be fixed here: https://github.com/GetStream/stream-chat-react/pull/283/commits/8ebde1ebd291c811cbcd57afaf05fd4a91a762db#diff-f031ec7f40f462e71c8f6641a01aa62fR245

File: src/components/Message/__tests__/MessageCommerce.test.js (hunk 1 above)
Comment (vicnicius) [reply src/components/Message/__tests__/MessageCommerce.test.js:202]: I'm going to merge as is to unblock some other PRs and introduce the generators later on a separate PR.

**Pull request merged**